Adamant/Modules/AlertsFactory/FreeTokensAlertManager.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  FreeTokensAlertManager.swift
+//  Adamant
+//
+//  Created by Владимир Клевцов on 23.1.25..
+//  Copyright © 2025 Adamant. All rights reserved.
+//
+import UIKit
+import SafariServices
+import CommonKit
+
+enum AlertPresenter {
+    @MainActor
+    static func freeTokenAlertIfNeed(type: FreeTokensAlertType) {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
```

Comment:
Maybe we can make AlertPresenter as injectable dependency? Register it in Swinject and it should be easily fetched from Swinject container anywhere across the project?